test/functional/feature_proxy.py

```diff
@@ -261,7 +269,7 @@ def networks_dict(d):
         assert_equal(n1['i2p']['proxy_randomize_credentials'], False)
         assert_equal(n1['i2p']['reachable'], True)
 
-        n2 = networks_dict(self.nodes[2].getnetworkinfo())
+        n2 = networks_dict(nodes_network_info[2])
```

Comment:
Approach ACK, is there a reason to not use `nodes_network_info` for `n3` and `n4`?
```diff
@ -285,7 +285,7 @@ class ProxyTest(BitcoinTestFramework):
         assert_equal(n2['cjdns']['reachable'], False)
 
         if self.have_ipv6:
-            n3 = networks_dict(self.nodes[3].getnetworkinfo())
+            n3 = networks_dict(nodes_network_info[3])
             assert_equal(NETWORKS, n3.keys())
             for net in NETWORKS:
                 if net == NET_I2P or net == NET_ONION:
@@ -298,7 +298,7 @@ class ProxyTest(BitcoinTestFramework):
             assert_equal(n3['i2p']['reachable'], False)
             assert_equal(n3['cjdns']['reachable'], False)
 
-        n4 = networks_dict(self.nodes[4].getnetworkinfo())
+        n4 = networks_dict(nodes_network_info[4])
         assert_equal(NETWORKS, n4.keys())
         for net in NETWORKS:
             if net == NET_I2P:
```